examples/19-EnsembleModel-examples.ipynb

```diff
@@ -0,0 +1,868 @@
+{
```

Comment:
<p>we could use the same time range as in the previous plot so that the visual comparison becomes easier</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/unit8co/darts/pull/1815/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>